server/models/schema.sql

```diff
@@ -1315,6 +1323,8 @@ CREATE TABLE `posting_journal` (
   KEY `period_id` (`period_id`),
   KEY `currency_id` (`currency_id`),
   KEY `user_id` (`user_id`),
+  KEY `cost_center_id` (`cost_center_id`),
```

Comment:
Will it be possible to rename this property fee_center_id or cost_profit_center_id